events.js

```diff
@@ -231,9 +231,11 @@ function changeDefaultBeacon(module) {
 // Triggered when the default beacon count is changed.
 function changeDefaultBeaconCount(event) {
     var count = RationalFromString(event.target.value)
-    spec.setDefaultBeacon(spec.defaultBeacon, count)
-    recipeTable.updateDisplayedModules()
-    itemUpdate()
+    if (count >= 0) {
```

Comment:
count is a Rational. This is not the correct comparison. It should be `if (!count.less(zero)) {`.

A corresponding check appears to be missing from BeaconCountHandler, if this check is even necessary, given the addition of the "min" attribute to the HTML elements.